hw1/02-palindrome.js

```diff
@@ -1,3 +1,40 @@
-const elem = document.querySelector('input');
+const integerInputElement = document.getElementById("integer_input");
+const resultElement = document.getElementById("result");
 
-elem.addEventListener('input', handleInput);
+integerInputElement.addEventListener("input", handleInput);
+
+function handleInput() {
+  let value = integerInputElement.value;
+  if (value < 0) {
+    resultElement.innerHTML = `<p style="color:red">Enter a positive number</p>`;
+  } else if (value.length === 0) {
+    resultElement.innerHTML = "";
+  } else if (isAlphabetsPresent(value)) {
+    resultElement.innerHTML = `<p style="color:red">Please enter a number.</p>`;
+  } else if (isPalindrome(value)) {
+    resultElement.innerHTML = `<p style="color:green">Yes. This is a palindrome!</p>`;
+  } else {
+    resultElement.innerHTML = `<p style="color:red">No. Try again.</p>`;
```

Comment:
I have never used the 'innerHTML' object but this is interesting. This definitely cleans up your HTML code but makes your JS file more verbose, guess it's just up to personal preference but I am definitely going to look into innerHTML in the future for my projects.

Reply:
Noted! Thanks for the review, may be next time I will take this into consideration and try to keep my JS file less verbose.